src/main/java/frc/robot/subsystems/ArmPivot.java

```diff
@@ -51,11 +105,35 @@ public void factoryDefaults() {
     configuration.MotorOutput.NeutralMode = NeutralModeValue.Brake;
     configuration.ClosedLoopGeneral.ContinuousWrap = true;
     cfg.apply(configuration);
-    // enabling stator current limits
+    // enabling current limits
     currentLimits.StatorCurrentLimit = 5; // starting low for testing
     currentLimits.StatorCurrentLimitEnable = true;
     currentLimits.SupplyCurrentLimit = 5; // starting low for testing
     currentLimits.SupplyCurrentLimitEnable = true;
     cfg.apply(currentLimits);
+
+    // PID
+    var slot0Configs = new Slot0Configs();
+    // set slot 0 gains
+    slot0Configs.kS = ARMPIVOT_KS; // Add 0.25 V output to overcome static friction
+    slot0Configs.kV = ARMPIVOT_KV; // A velocity target of 1 rps results in 0.12 V output
+    slot0Configs.kA = ARMPIVOT_KA; // An acceleration of 1 rps/s requires 0.01 V output
+    slot0Configs.kP = ARMPIVOT_KP; // A position error of 2.5 rotations results in 12 V output
+    slot0Configs.kI = ARMPIVOT_KI; // no output for integrated error
+    slot0Configs.kD = ARMPIVOT_KD; // A velocity error of 1 rps results in 0.1 V output
+    slot0Configs.kG = ARMPIVOT_KG;
```

Comment:
All of the other variables have very clear comments describing what they are.  (Good job!)  You might add a comment for this one too.